packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message-content.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { getBrandName } from '@deriv/shared';
+import CorrectMessageContent from './correct-message-content';
+import FakeMessageContent from './fake-message-content';
+
+const brandName = getBrandName();
+
+const WarningScamMessageContent = ({ is_dark_mode_on }) => (
```

Comment:
@suisin-deriv Just a question, Props like this `is_dark_mode_on` are in the store which the component can access directly, So why still prop drilling? 🤔
Same question for other components like `WarningScamMessageModal` 🤔